cmd/kubeapply/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/datawire/teleproxy/pkg/kubeapply"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func envBool(name string) bool {
+	val := os.Getenv(name)
+	switch strings.TrimSpace(strings.ToLower(val)) {
+	case "true":
+		return true
+	case "yes":
+		return true
+	case "1":
+		return true
+	case "false":
+		return false
+	case "no":
+		return false
+	case "0":
+		return false
+	case "":
+		return false
+	}
+
+	return true
+}
```

Comment:
Is there a reason you didn't use `strconv.ParseBool()`?

Reply:
No clue, that code is from ages ago. Maybe I didn't know it existed. Thanks for the pointer. ;-)